org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,12 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.compliance=1.8
+org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enablePreviewFeatures=disabled
+org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.problem.forbiddenReference=warning
+org.eclipse.jdt.core.compiler.problem.reportPreviewFeatures=ignore
+org.eclipse.jdt.core.compiler.problem.unqualifiedFieldAccess=warning
```

Comment:
@ghislainfourny This compiler option tunes behavior for unqualified access to class fields, ie. using myField instead of this.myField. I have used this in the error mode to add consistency to the entire project and refactored all field usages to be a qualified field access. However, the autogenerated parser file is filled with thousands of unqualified accesses. I am yet to find an option to disable this check on a file, so for now I have downgraded the error to a warning, so that the program can compile and tests can be executed on it.

For future usage, we have a few options, we either:
- Keep it on warning and keep an eye out for compiler warnings
- Hopefully find a way to disable this option on a file
- Or tweak either our usage of ANTLR tool or implement a postprocessing script to sanitize it from unqualified accesses so that we can go back to using errors for unqualifies accesses.